metaflow/plugins/argo/argo_workflows_cli.py

```diff
@@ -541,3 +542,100 @@ def _convert_value(param):
             "See the run in the UI at %s" % run_url,
             bold=True,
         )
+
+
+@argo_workflows.command(help="List all runs of the workflow on Argo Workflows.")
```

Comment:
nit - here and below - we can specify flow instead of workflow since workflow has a very different connotation in Argo Workflows.